test/e2e/framework/framework.go

```diff
@@ -333,7 +333,7 @@ func (f *Framework) AfterEach() {
 		if len(nsDeletionErrors) != 0 {
 			messages := []string{}
 			for namespaceKey, namespaceErr := range nsDeletionErrors {
-				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s", namespaceKey, namespaceErr))
+				messages = append(messages, fmt.Sprintf("Couldn't delete ns: %q: %s (%#v)", namespaceKey, namespaceErr, namespaceErr))
```

Comment:
That's the bug - why do we have an aggregate of length 1?  It should
be unwrapped, because NotFound will fail on aggregates.